app/components/bulk_upload_error_summary_table_component.html.erb

```diff
@@ -1,24 +1,16 @@
-<%= govuk_table do |table| %>
-  <% table.caption(size: "m", text: bulk_upload.filename) %>
-
-  <% table.head do |head| %>
-    <% head.row do |row| %>
-      <% row.cell(text: "Column", header: true) %>
-      <% row.cell(text: "Number of rows", header: true) %>
-      <% row.cell(text: "Question", header: true) %>
-      <% row.cell(text: "Error", header: true) %>
-      <% row.cell(text: "Specification", header: true) %>
-    <% end %>
-  <% end %>
+<% sorted_errors.each do |error| %>
+  <%= govuk_table do |table| %>
+    <% table.head do |head| %>
+      <% head.row do |row| %>
+        <% row.cell(text: question_for_field(error[0][1].to_sym), header: true) %>
```

Comment:
What's `error[0][1]` and below `error[0][2]` in this instance?

Reply:
sorry it's not great. it because its doing an aggregate in sql so the output is something like
```ruby
[
  [["H", "field_7", "This is a duplicate log"], 9],
  [["M", "field_12", "This is a duplicate log"], 9],
  ...
]
```
the aggregate is the count or in this case `9` in the above example the other part is the group by part